adapters/adapters-commons/src/test/java/uk/gov/justice/raml/common/validator/RequestContentTypeRamlValidatorTest.java

```diff
@@ -54,35 +55,35 @@ public void shouldThrowExceptionIfMediaTypeNotSet() throws Exception {
 
         validator.validate(raml()
                 .with(resource()
-                        .with(action().withActionType(POST)))
+                        .with(httpAction().withHttpActionType(POST)))
                 .build());
 
     }
 
     @Test
-    public void shouldThrowExceptionIfMediaTypeDoesNotContainAValidCommand() throws Exception {
+    public void shouldThrowExceptionIfMediaTypeDoesNotContainFormat() throws Exception {
 
         exception.expect(RamlValidationException.class);
-        exception.expectMessage("Invalid request type: application/vnd.people.invalid.command1+json");
+        exception.expectMessage("Invalid request type: application/vnd.command1");
 
         validator.validate(
                 raml()
                         .with(resource()
-                                .with(action(POST, "application/vnd.people.invalid.command1+json")))
+                                .with(httpAction(POST, "application/vnd.command1")))
                         .build());
 
     }
 
     @Test
-    public void shouldThrowExceptionIfNotvalidMediaType() throws Exception {
+    public void shouldThrowExceptionIfNotAVendorMediaType() throws Exception {
```

Comment:
Should we have a test for something like +xml to make it clear that the ActionMapping class is hard-coded to only work with +json and would need a change to support any other set of media types?